hp-ux/generate_wazuh_packages.sh

```diff
@@ -169,8 +169,11 @@ clean() {
     ${install_path}/bin/${control_binary} stop
   fi
 
+  if [ -f /etc/ossec-init.conf ]; then
```

Comment:
I think we don't need to check ossec-init.conf here. Because the file will be removed during an upgrade and will not be installed enymore.

Reply:
Fixed